packages/experimental/ExperimentalIcon/src/Icon.types.ts

```diff
@@ -0,0 +1,8 @@
+import { SvgIconProps } from './SvgIcon/SvgIcon.types';
+import { FontIconProps } from './FontIcon/FontIcon.types';
+export const icon = 'Icon';
```

Comment:
What's this const for? Is it supposed to be the name?

Reply:
Yes, it, also, was generated. I'll remove it because we have these names for FontIcon and SVGIcon. Thank you for catching this. And yes, you're right, it's better to split such PRs because I thought I checked everything before check-in but looks like I missed things like that